tsconfig.json

```diff
@@ -10,11 +10,12 @@
     "allowSyntheticDefaultImports": true,
     "experimentalDecorators": true,
     "importHelpers": true,
-    "outDir": "dist",
+    "outDir": "build",
```

Comment:
Question: why the directory change?

Reply:
since we are now using vite for bundling to prod-ish and development, i figured we can use the same directory for it, but we could change it to be both `dist` instead if we want to

Reply:
I am not familiar with the interaction of build tools enough to know what to check for. 

For example, in `tsconfig.json` we have: 
```
{
  "compilerOptions": {
     "outDir": "dist",
     ...
```
Are we going to have both directories?

Reply:
I also see a failing build step at: 
```
Error: Could not find any test files with pattern(s): dist/test/**/*.test.js
```

Reply:
seems we need both directories, `build` for gh-pages publishing; `dist` for tests

Reply:
yup, moved now